torch_glow/src/PyTorchCommon.h

```diff
@@ -40,6 +40,11 @@ PyTorchLoaderSettings &getPyTorchLoaderSettings();
 /// \returns the PyTorch symbol to be used for the PyTorch node which represents
 /// the subgraph that Glow will compile and run.
 const c10::Symbol &getGlowSymbol();
+
+/// Executes custom fuse pass for the given \p graph and \p fuseSymbol.
+void glowCustomFuse(std::shared_ptr<torch::jit::Graph> &graph,
```

Comment:
where is the thread_local stuff documented?

Reply:
Nowhere, this is hidden from public implementation details, caller shoudn't even know about all this local thread stuff. What caller needs to know is the ability to invoke public methods of PyTorchFileLoader class in multiple threads.

Reply:
discussed offline, makes sense (I think)